static/js/zamboni/buttons.js

```diff
@@ -179,16 +178,28 @@ var installButton = function() {
         };
     });
 
+    var showDownloadAnyway = function($button) {
+        var $installShell = $button.parents('.install-shell');
+        var $downloadAnyway = $installShell.find('.download-anyway');
+        if ($downloadAnyway.length) {
+            var href = $button.filter(':visible').attr('href');
+            var pos = href.lastIndexOf('/');
+            href = escape_(href.slice(0, pos) + '/type:attachment/' + href.slice(pos + 1));
```

Comment:
It should be the `downloads.latest` URL, but I forgot platforms... I'll check locally with multi-platforms add-ons.